File: LC485.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        c = 0
        m = 0

        for i in nums:
            if i:
                c += 1
                m = max(m, c)
            else:
                c = 0
        
        return m

File: LC485.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int c = 0, m = 0;

        for(auto i : nums)
            if(i)
                m = max(m, ++c);
            else
                c = 0;
        
        return m;
    }
};
